executor/slow_query.go

```diff
@@ -198,6 +201,8 @@ func (e *slowQueryRetriever) getPreviousFile() *os.File {
 }
 
 func (e *slowQueryRetriever) parseDataForSlowLog(ctx context.Context, sctx sessionctx.Context) {
+	e.wg.Add(1)
```

Comment:
```Add``` should put at the usage of ```parseDataForSlowLog```.


···
e.wg.Add(1)
go e.parseDataForSlowLog(ctx, sctx)
···

BTW can you use the ```util.WaitGroupWrapper``` ？